src/udiskslinuxblock.c

```diff
@@ -2988,25 +2989,27 @@ udisks_linux_block_handle_format (UDisksBlock             *block,
   /* And now create the desired filesystem */
   wait_data->type = type;
 
+  device_name = udisks_block_dup_device (block);
```

Comment:
We probably should replace the "wipefs" call above (I can't add a comment to unchanged lines) with libblockdev.